spalp_example/spalp_example.config.schema.json

```diff
@@ -0,0 +1,72 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "config": {
+      "type": "object",
+      "properties": {
+        "apiKey": {
+          "type": "integer"
+        },
+        "endpoint": {
+          "type": "string"
+        },
+        "nestedConfig": {
+          "type": "object",
+          "properties": {
+            "firstParam": {
+              "type": "string"
+            },
+            "secorndParam": {
+              "type": "boolean"
+            }
+          },
+          "required": [
+            "firstParam",
+            "secorndParam"
+          ]
+        }
+      },
+      "required": [
+        "apiKey",
+        "endpoint",
+        "nestedConfig"
+      ]
+    },
+    "app_text": {
+      "type": "object",
+      "properties": {
+        "firstField": {
```

Comment:
ideally we'd have meaningful field names that match the existing config.json file
https://github.com/Capgemini/spalp/blob/8.x-1.x/spalp_example/spalp_example.config.json